packages/ui/src/components/va-input/_variables.scss

```diff
@@ -43,4 +43,16 @@
   --va-input-container-label-font-weight: 700;
   --va-input-container-label-text-transform: uppercase;
   --va-input-container-label-transform-origin: top left;
+
+  /* Bottom */
+  --va-input-wrapper-bottom-display: flex;
```

Comment:
Does it make sense to have display types in variables?

Looks like something very fragile: user might shoot their foot or we might screw this over on vuestic update.
Do we use variables like this elsewhere?

Reply:
> Do we use variables like this elsewhere?
Yes, it's pretty common.